PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+### What PR does?
```

Comment:
**What this PR does?** instead of **WHat PR does?**

Reply:
@katunold this has been corrected